src/main/java/io/lenses/connect/smt/header/Constants.java

```diff
@@ -0,0 +1,7 @@
+package io.lenses.connect.smt.header;
+
+import java.time.ZoneId;
+
+public class Constants {
+  public static final ZoneId UTC = ZoneId.of("UTC");
```

Comment:
It could be `public static final ZoneId UTC = ZoneOffset.UTC.normalized();` but I like it, it's verbose